### PR TITLE
[TEST] MPA unit tests for modules/classifiers

### PR DESCRIPTION
**Pull request by kprokofi**
image

### CODE REVIEW

**Review by sovrasov — DISMISSED** [review]

**Review by sungmanc — DISMISSED** [review]

**Review by jaegukhyun — COMMENTED** [review]
Test codes and changes in otx look good to me, but there are nncf errors in classification task.
Could you take a look?

File: tests/unit/mpa/modules/models/classifiers/test_sam_classifier.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                "some_prefix.act.weight": torch.rand(1),
+                "some_prefix.someweights": torch.rand(2),
+            }
+        elif name == "effnetv2":
+            self.backbone = OTXEfficientNetV2()
+            self._state_dict = {
+                "some_prefix.model.classifier.weight": torch.rand(1, 1),
+                "some_prefix.model.weight": torch.rand(1),
+            }
+        else:
```
Comment (JihwanEom) [inline tests/unit/mpa/modules/models/classifiers/test_sam_classifier.py]: How about `elif name == "effnet"`

**Review by JihwanEom — COMMENTED** [review]
I left minor comment, thank you for hard work and sorry for complicated structure for `SAMClassifier` :(

**Comment by JihwanEom** [comment]
> Test codes and changes in otx look good to me, but there are nncf errors in classification task. Could you take a look?

I agree with Jaeguk, `model_name = prefix + model_name` seems like a troublemaker.

**Comment by goodsong81** [comment]
Can I merge this PR, or need to rebase & run integration tests again?

**Review by sovrasov — APPROVED** [review]

**Review by goodsong81 — APPROVED** [review]
LGTM!

**Review by JihwanEom — APPROVED** [review]

**Pull request merged**